.github/workflows/elixir.yml

```diff
@@ -2,56 +2,59 @@ name: Elixir CI
 
 on:
   push:
-  pull_request:
     branches:
       - main
-
-env:
-  ELIXIR_LATEST: '1.16.0'
+  pull_request:
 
 permissions:
   contents: read
 
 jobs:
   build:
-    name: Build and test with elixir ${{ matrix.elixir }} otp ${{ matrix.otp }}
     runs-on: ubuntu-20.04
     strategy:
       matrix:
         include:
-          - elixir: '1.14.1'
-            otp: '25.0'
-          - elixir: '1.15.0'
-            otp: '26.0'
-          - elixir: '1.16.0'
-            otp: '26.0'
+          - elixir: '1.14'
+            otp: '25'
+          - elixir: '1.17'
+            otp: '27'
+            lint: true
     steps:
```

Comment:
I'd rather keep testing 15 and 16, too. Also I prefer specifying the minor version (I chose 1.14.1 because LV 1.0 isn't compatible with 1.14.0) in order to have predictable runs. If we only specify 1.x, which minor version will be picked?

Reply:
I will enable it back. When we specify on the major version, `1.14`, `setup-beam` will use the latest patch Elixir version of `1.14.5`